neural_modelling/src/delay_extension/Makefile

```diff
@@ -3,3 +3,5 @@ BUILD_DIR = build/
 SOURCES = delay_extension.c
 
 include ../common.mk
+
+CFLAGS += -I $(NEURAL_MODELLING_DIRS)/src/common
```

Comment:
I believe that there shouldn't usually be a space between `-I` and the directory

Reply:
Our pattern is to include the space

See:  lines 54 and 55 in
https://github.com/SpiNNakerManchester/spinnaker_tools/blob/master/make/Makefile.common

The space may not be required but that is out of scope for this PR